conf/machine/orangepi-i96.conf

```diff
@@ -0,0 +1,40 @@
+#@TYPE: Machine
+#@NAME: OrangePi i96 machine
+#@DESCRIPTION: OrangePi i96 machine
+
+require conf/machine/include/tune-cortexa5.inc
+
+PREFERRED_PROVIDER_virtual/xserver ?= "xserver-xorg"
+
+XSERVER ?= "xserver-xorg \
+            mesa-driver-swrast \
+            xf86-input-evdev \
+            xf86-video-fbdev"
+
+MACHINE_FEATURES = "usbhost usbgadget alsa screen wifi bluetooth ext2"
+
+PREFERRED_PROVIDER_virtual/kernel ?= "linux-orangepi-i96"
+
+KERNEL_IMAGETYPE = "zImage"
+#KERNEL_DEVICETREE = "currently-uses-a-concatenated-dt"
+
+UBOOT_MACHINE = "rda8810_config"
+UBOOT_SUFFIX = "rda"
+
+# TODO: Switch this to LS-UART1 and reduce to 115200 (921600 is needed
+#       on ttyS0 in order to match speeds with the boot ROM output
+SERIAL_CONSOLE = "921600 ttyS0"
```

Comment:
The SERIAL_CONSOLE variable is deprecated. Please use the SERIAL_CONSOLES variable. The format is slightly different, it isn't only a renaming of the variable.